src/benchmarks/gc/README.md

```diff
@@ -187,6 +192,34 @@ Each trace file will be named `{coreclr_name}__{config_name}__{benchmark_name}__
 
 _ARM NOTE_: Container tests are not supported on ARM/ARM64.
 
+### Running the Entire Suite
```

Comment:
I would definitely put this as the first thing in the "Running" section and mentioned that this is what you should always run to make sure you don't regress perf if you are making a perf change in GC.

Reply:
Sounds good. I also forgot to add that to the `Contributing` doc. Will do that as well.